layouts/shortcodes/page/about.html

```diff
@@ -0,0 +1,99 @@
+<div id="prowered-by" class="row featured-section featured-section-prefix text-center">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-14 col-md-offset-5 col-sm-18 col-sm-offset-3">
+                <p class="small">OUR MISSION</p>
+                <p class="powered-by-description">To define and build an ecosystem of best-in-class open-source web and
+                    cloud-based development tools, and to promote and drive the broad adoption of these tools.</p>
+                <p><a class="btn btn-primary" href="https://accounts.eclipse.org/mailing-list/ecd-tools-wg">Get Involved</a></p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="row featured-section featured-content-about">
+    <div class="container">
+        <div class="row margin-bottom-30">
+            <div id="working_group" class="col-sm-18 col-sm-offset-3">
+                <p class="margin-bottom-30 text-center"><img class="featured-content-about-img img-responsive" src="/images/ecdtools/about/about-icon.png"
+                        alt="ABOUT THE ECLIPSE CLOUD DEVTOOLS WORKING GROUP"></p>
```

Comment:
Alt text doesn't need capitalization, and may act weird on some screen readers in this state